modules/objdetect/include/opencv2/objdetect.hpp

```diff
@@ -693,23 +693,43 @@ class CV_EXPORTS_W QRCodeDetector
      */
     CV_WRAP bool detect(InputArray img, OutputArray points) const;
 
+    /** @brief Detects QR codes in image and returns the vector of the quadrangle containing the codes.
+     @param img grayscale or color (BGR) image containing (or not) QR codes.
+     @param points Output vector of vector of vertices of the minimum-area quadrangle containing the codes.
+     */
+    CV_WRAP bool detectMulti(InputArray img, OutputArrayOfArrays points) const;
+
     /** @brief Decodes QR code in image once it's found by the detect() method.
      Returns UTF8-encoded output string or empty string if the code cannot be decoded.
-
      @param img grayscale or color (BGR) image containing QR code.
      @param points Quadrangle vertices found by detect() method (or some other algorithm).
      @param straight_qrcode The optional output image containing rectified and binarized QR code
      */
     CV_WRAP cv::String decode(InputArray img, InputArray points, OutputArray straight_qrcode = noArray());
 
-    /** @brief Both detects and decodes QR code
+    /** @brief Decodes QR codes in image once it's found by the detect() method.
+     Returns UTF8-encoded output vector of string or empty vector of string if the codes cannot be decoded.
+     @param img grayscale or color (BGR) image containing QR codes.
+     @param points vector of Quadrangle vertices found by detect() method (or some other algorithm).
+     @param straight_qrcode The optional output vector of images containing rectified and binarized QR codes
+     */
+    CV_WRAP std::vector<cv::String> decodeMulti(InputArray img, InputArrayOfArrays points, OutputArrayOfArrays straight_qrcode = noArray());
```

Comment:
We usually don't return `std::vector` from functions (it is very inefficient in C++98):
```
bool decodeMulti(InputArray img, CV_OUT std::vector<std::string>& decoded_info,
        InputArrayOfArrays points, OutputArrayOfArrays straight_qrcode = noArray());
```

`cv::String` => `std::string`

Reply:
@alalek  std::vector<std::string> doesn't work fine in python and java
Maybe, it is better to use std::vector<cv::String> to avoid problems?